includes/class-wcpdf-frontend.php

```diff
@@ -99,31 +100,41 @@ public function woocommerce_api_invoice_number ( $data, $order ) {
 		return $data;
 	}
 
-	/**
-	 * Download invoice frontend shortcode
-	 */
-	public function download_invoice_shortcode( $atts ) {
+	public function generate_document_link_shortcode( $atts ) {
 		global $wp;
 
 		if ( is_admin() ) {
 			return;
 		}
 
 		// Default values
-		$values = shortcode_atts (array(
-			'order_id'		=> '',
-			'link_text'		=> ''
+		$values = shortcode_atts( array(
+			'order_id'      => '',
+			'link_text'     => __(
+				'Download ' . str_replace( '-', ' ', $atts['document_type'] ?? 'invoice' ) . ' (PDF)',
```

Comment:
Sorry, I think this translation string must be improved, is not easy to understand for translators. Maybe you should use a `sprintf` with a placeholder and also add the translators help comment, like this:

https://github.com/wpovernight/woocommerce-pdf-invoices-packing-slips/blob/bd60be023109387b4eb34c01ca2a7b9ee7115613/includes/class-wcpdf-admin.php#L715-L718

Reply:
Sure, no problem.